nova3/engines/eztv.py

```diff
@@ -61,7 +61,7 @@ def handle_endtag(self, tag):
 
     def search(self, what, cat='all'):
         query = self.url + '/search/' + what.replace('%20', '-')
-        eztv_html = retrieve_url(query)
+        eztv_html = retrieve_url(query, post_data=b"layout=def_wlinks")
```

Comment:
Ain't this is going to break with existing installations?
Changes in https://github.com/qbittorrent/qBittorrent/pull/21184 won't reach users immediately.

Reply:
It will, hence why I suggested maybe I should bundle the updated `retrieve_url` function in eztv.py.

I haven't done so yet because I wanted maintainers to decide the best course of action in this situation.

Reply:
Whatever you do, just don't break it. It is the only thing that matters.